builds/misc/packages.yaml

```diff
@@ -135,7 +135,13 @@ jobs:
     steps:
       - powershell: |
           $base_version = Get-Content -Path "$(Build.SourcesDirectory)\edgelet\version.txt"
-          $version = ("{0}{1}" -f $base_version, $(Build.BuildNumber))
+          if ($base_version -like '*~*') {
+            $version = ("{0}{1}" -f $base_version, $(Build.BuildNumber))
+          } else {
+            $version = $base_version
+          }
+          echo "base_version=$base_version"
```

Comment:
Are you planning to leave these in or are they just for debugging? If you're planning to leave them in, use `Write-Host` instead of `echo`.

Reply:
Done.